lib/handlers/compile.js

```diff
@@ -154,7 +154,7 @@ class CompileHandler {
     }
 
     parseRequest(req, compiler) {
-        let source, options, backendOptions, filters, bypassCache = false;
+        let source, options, backendOptions, filters, bypassCache = false, tools;
```

Comment:
Should this be noted somewhere on the api docs?

Reply:
Definitely. I think it might be a good idea to extract the API texts from the readme.md into a separate file, and also add all the new things we added recently.